src/main/java/de/teamlapen/vampirism/tileentity/TotemTileEntity.java

```diff
@@ -622,14 +624,14 @@ private void setCapturingFaction(@Nullable IFaction faction) {
         this.progressColor = faction != null ? faction.getColor().getColorComponents(null) : DyeColor.WHITE.getColorComponentValues();
     }
 
-    public @Nonnull
-    AxisAlignedBB getVillageArea() {
-        return this.villageArea == null ? this.villageArea = AxisAlignedBB.func_216363_a(this.village.getBoundingBox()) : this.villageArea;
+    @Nonnull
+    public AxisAlignedBB getVillageArea() {
+        return this.villageArea == null ? this.village != null ?this.villageArea = AxisAlignedBB.func_216363_a(this.village.getBoundingBox()) :new AxisAlignedBB(0,0,0,0,0,0): this.villageArea;
     }
 
-    private @Nonnull
-    AxisAlignedBB getVillageAreaReduced() {
-        return this.villageAreaReduced == null ? this.villageAreaReduced = AxisAlignedBB.func_216363_a(this.village.getBoundingBox()).grow(-30, -10, -30) : this.villageAreaReduced;
+    @Nonnull
+    private AxisAlignedBB getVillageAreaReduced() {
```

Comment:
it is possible that the entity ai task was updated before the world was even loaded ?! therefore a check